CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 - Added Crons support via `SentrySdk.CaptureCheckIn` and an integration with Hangfire ([#3128](https://github.com/getsentry/sentry-dotnet/pull/3128))
 
+### API changes
+
+- Removed `ScopeExtensions` class - all the public methods moved directly to `Scope` ([#3186](https://github.com/getsentry/sentry-dotnet/pull/3186))
```

Comment:
Is that change even visible to users?

Reply:
Shouldn't be for c# users

Reply:
Their code won't have to change if they're recompiling with the new code. Technically I guess it's not binary compatible (if someone was using reflection to get at these extension methods for some strange reason, that would break)... I'm not sure how hard core we want to be about avoiding breaking changes. @bruno-garcia can we be "pragmatic" here?

Reply:
> can we be "pragmatic" here?

Yes, especially since this is improving/fixing things on the Powershell side.